src/class/cdc/cdc_host.c

```diff
@@ -433,6 +433,77 @@ bool tuh_cdc_read_clear (uint8_t idx) {
 // Control Endpoint API
 //--------------------------------------------------------------------+
 
+// set line coding using sequence with 2 stages: set baudrate (stage1) + set data format (stage2)
```

Comment:
Here is the central 2-stage-sequence of setting baudrate and data format outsourced from CH34x into common used set_line_coding_sequence() and set_line_coding_stage1_complete().
It will be also used by FTDI's and and CP210x's set line coding functions (see next commits), because they also requires this sequence.